src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -19,7 +20,11 @@
     'customHeaders': 'HEADERS',
     'limit': 'LIMIT',
     'currentValue': 'CURRENT VALUE',
-    'unit': 'UNIT'
+    'unit': 'UNIT',
+    'buildId': "BUILDID",
+    #'trigger': "TRIGGER",
```

Comment:
Why is this commented out?

Reply:
Because we are still waiting for the unified schema of the json for all three builds, including git, manual and base.